Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -646,139 +646,139 @@ Don't have a 4chan Pass?<br>
-    <string name="report_controller_logs_are_empty_error">\"Attach logs\" option is checked, but logs are empty!</string>
-    <string name="report_controller_report_sent">Sent report!</string>
-    <string name="report_controller_error_while_trying_to_send_report">Error while trying to send report: %1$s</string>
+    <string name="report_controller_title_cannot_be_empty_error">Supply a title first.</string>
+    <string name="report_controller_description_cannot_be_empty_error">If the description is empty, you have to supply a log.</string>
+    <string name="report_controller_logs_are_empty_error">You have \"Attach logs\" checked, but no logs attached!</string>
+    <string name="report_controller_report_sent">Report sent.</string>
+    <string name="report_controller_error_while_trying_to_send_report">Could not send report: %1$s</string>
 
     <string name="hsid_cookie_text">HSID</string>
     <string name="ssid_cookie_text">SSID</string>
     <string name="sid_cookie_text">SID</string>
     <string name="nid_cookie_text">NID</string>
     <string name="js_captcha_cookies_editor_format_hint">Do not prepend values with \"HSID=…\", \"SSID=…\" etc.</string>
-    <string name="settings_js_captcha_cookies_title">JS captcha cookies</string>
-    <string name="settings_js_captcha_cookies_description">By inputting your desktop browser captcha cookies, captcha solving may become easier</string>
-    <string name="cookies_editor_bad_cookie">%1$s cookie is not set!</string>
-    <string name="cookies_editor_failed_parse">Failed to parse cookies, re-enter them correctly!</string>
+    <string name="settings_js_captcha_cookies_title">JavaScript CAPTCHA cookies</string>
+    <string name="settings_js_captcha_cookies_description">By inputting CAPTCHA cookies from your desktop web browser, solving CAPTCHAs may become easier.</string>
+    <string name="cookies_editor_bad_cookie">%1$s cookie not set!</string>
+    <string name="cookies_editor_failed_parse">Could not parse cookies, re-enter them correctly.</string>
 
     <string name="settings_report">Report</string>
-    <string name="settings_report_description">Report a problem/crash</string>
+    <string name="settings_report_description">Report a problem or crash.</string>
     <string name="settings_collect_crash_logs">Collect crash logs</string>
-    <string name="settings_collect_crash_logs_description">By enabling this setting, the app will start collecting crash logs. They are NOT uploaded automatically. Crash logs only collect the stacktrace itself, app version and basic OS information. You can review and upload them manually (or delete) by clicking the \"Report\" setting.</string>
+    <string name="settings_collect_crash_logs_description">Starts collecting app-crash logs (stacktrace, app version and basic OS info), to be reviewed and sent (from \"Report\") if you wish to do so.</string>
```

Comment:
Prefer the start of your change, but then falls apart at the second part.

Reply:
```suggestion
    <string name="settings_collect_crash_logs_description">Starts collecting app-crash logs (stacktrace, app version and basic OS info), to be review and optionally send from \"Report\".</string>
```